### PR TITLE
Update readme and labels

### PR DESCRIPTION
**Pull request by jgunnink**
- Lables now are more descriptive in describing what the function does
- Readme is re-organised to be clearer and shows the steps in chronological order
- `vendor` added to gitignore

### CODE REVIEW

File: cmd/gen.go [hunk 1]
```diff
@@ -11,13 +11,12 @@ import (
 // genCmd represents the gen command
 var genCmd = &cobra.Command{
 	Use:   "gen",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
+	Short: "generates the letter template",
```
Comment (nii236) [inline cmd/gen.go]: Start this with a capital please

File: cmd/gen.go [hunk 2]
```diff
@@ -11,13 +11,12 @@ import (
...
+	Long: `Gen creates a letter to a reciepient and accepts flags including the
```
Comment (nii236) [inline cmd/gen.go]: recipient

**Comment by nii236** [comment]
👍

File: cmd/gen.go (hunk 2 above)
Comment (jgunnink) [inline cmd/gen.go]: 10 points! You found my trap 👍

**Pull request merged**